common/maplib/map.mjs

```diff
@@ -438,6 +470,29 @@ class _SingleGameMap {
     if (tile === null) return null;
     return this.graphicAsset.getTile(layer, tile);
   }
+
+  /**
+   * Get spawn points of single map.
+   * If We have a spawn point area from (1,1) (1,5) (5,1) (5,5),
+   * we split it here to become 16 spawn points.
+   * @return {Array} The map coordinates inside 'spawnPoint' cell set.
+   */
```

Comment:
This PR contains 3 types of [JSDoc](https://jsdoc.app/about-block-inline-tags.html).

```js
/**
 * First
**/
 ```

```js
/**
 * First
 **/
 ```

```js
/**
 * First
 */
 ```